packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,45 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+
+  &.typeSecondary {
+    background: none;
```

Comment:
Since we're removing background here, should we add a `background-color: c.$colorNeutralWhite` to .wrapper? I am not sure

Reply:
@jozefhruska But does `primary` have no background as well?

Reply:
Yes, you're right. Thanks 🙏 Sorry I didn't understand your comment at first 😌